src/plugins/data_source/server/client/configure_client.ts

```diff
@@ -70,13 +78,18 @@ export const configureTestClient = async (
 export const getDataSource = async (
   dataSourceId: string,
   savedObjects: SavedObjectsClientContract
-): Promise<SavedObject<DataSourceAttributes>> => {
-  const dataSource = await savedObjects.get<DataSourceAttributes>(
+): Promise<DataSourceAttributes> => {
+  const dataSourceSavedObject = await savedObjects.get<DataSourceAttributes>(
     DATA_SOURCE_SAVED_OBJECT_TYPE,
     dataSourceId
   );
 
-  return dataSource;
+  const dataSourceAttr = {
+    ...dataSourceSavedObject.attributes,
+    lastUpdatedTime: dataSourceSavedObject.updated_at,
```

Comment:
reminder to also update type def for `DataSourceAttributes` with lastupdatedTime

Reply:
that's done